openid-4-verifiable-presentations-1_0.md

```diff
@@ -2797,6 +2810,7 @@ The technology described in this specification was made available from contribut
    * require `typ` value in request object to be `oauth-authz-req+jwt`
    * add `SessionTranscript` requirements 
    * use claims path pointer for mdoc based credentials
+   * support returning multiple presentations for a single dcql credential query when requested using `multiple`
```

Comment:
Shouldn't this be added to 25? as 24 has already been published? 
https://openid.net/specs/openid-4-verifiable-presentations-1_0-24.html

Reply:
If that's the case then it would probably be best if one of the maintainers should increment the version numbers rather than including it in this PR.